sentry-kotlin-multiplatform/src/commonMain/kotlin/io/sentry/kotlin/multiplatform/SentryScopeCallback.kt

```diff
@@ -0,0 +1,5 @@
+package io.sentry.kotlin.multiplatform
+
+fun interface SentryScopeCallback {
```

Comment:
Do we need to declare an interface?
Maybe we could use higher-order functions/callbacks.

Reply:
I'll replace it, it's actually better if we use higher-order functions since Kotlin - Obj-C conversion doesn't recognize these interfaces as callbacks